python/tests/test_metrics.py

```diff
@@ -502,34 +457,31 @@ def test_proto_seldon_metrics_predict(cls):
     )
 
     request = prediction_pb2.SeldonMessage(data=datadef)
-    app.Predict(request, None)
 
-    data = seldon_metrics.data[os.getpid()]
-    assert data["GAUGE", "mygauge"]["value"] == 100
-    assert data["GAUGE", "customtag"]["value"] == 200
-    assert data["GAUGE", "customtag"]["tags"] == {"mytag": "mytagvalue"}
-    assert data["COUNTER", "mycounter"]["value"] == 1
-    assert np.allclose(
-        np.histogram([20.2 / 1000], BINS)[0], data["TIMER", "mytimer"]["value"][0]
-    )
-    assert np.allclose(data["TIMER", "mytimer"]["value"][1], 0.0202)
-
-    app.Predict(request, None)
+    resp = app.Predict(request, None)
+    assert (
+        "metrics" in json.loads(json_format.MessageToJson(resp))["meta"]
+    ) == client_gets_metrics
+    assert (
+        "metrics" in json.loads(json_format.MessageToJson(resp))["meta"]
+    ) == client_gets_metrics
 
     data = seldon_metrics.data[os.getpid()]
-    assert data["GAUGE", "mygauge"]["value"] == 100
-    assert data["GAUGE", "customtag"]["value"] == 200
-    assert data["GAUGE", "customtag"]["tags"] == {"mytag": "mytagvalue"}
-    assert data["COUNTER", "mycounter"]["value"] == 2
-    assert np.allclose(
-        np.histogram([20.2 / 1000, 20.2 / 1000], BINS)[0],
-        data["TIMER", "mytimer"]["value"][0],
-    )
-    assert np.allclose(data["TIMER", "mytimer"]["value"][1], 0.0404)
+    verify_seldon_metrics(data, 1, [0.0202])
+
+    resp = app.Predict(request, None)
+    assert (
+        "metrics" in json.loads(json_format.MessageToJson(resp))["meta"]
+    ) == client_gets_metrics
+    assert (
+        "metrics" in json.loads(json_format.MessageToJson(resp))["meta"]
```

Comment:
Same as above, why are we asserting the same thing twice?

Reply:
yeah, just a mistake